make/run-cicd-script.mk

```diff
@@ -1,5 +1,5 @@
 
-OWNER_AND_BRANCH_LOCATION=codeready-toolchain/toolchain-cicd/master
+OWNER_AND_BRANCH_LOCATION=rajivnathan/toolchain-cicd/saveLogsToArtifactsDir
```

Comment:
?

Reply:
this is what Matous was suggesting to do to test changes from our toolchain-cicd repo. This PR will not be merged.